src/Components/Atoms/AuthButton/AuthButtonStyle.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const AuthBtn = styled.button`
+  padding: 10px 20px;
+  font-weight: bold;
+  border-radius: 10px;
+  border: none;
+  background-color: #dde3e7;
```

Comment:
Please read about creating global files / themes for colors and font sizes.
Its for defining all colors, fonts, font-size etc in one file and use it throughout the application.
No need to make this change in this PR but add this change from next assignments.